packages/amplication-client/src/VersionControl/CommitListItem.tsx

```diff
@@ -18,12 +18,12 @@ export const CommitListItem = ({ commit, projectId }: Props) => {
   const { currentWorkspace } = useContext(AppContext);
   const { commitStatus } = useCommitStatus(commit);
   return (
-    <div className={CLASS_NAME}>
+    <div className={`${CLASS_NAME}__row`}>
       <InnerTabLink
         icon=""
         to={`/${currentWorkspace?.id}/${projectId}/commits/${commit.id}`}
       >
-        <div className={`${CLASS_NAME}`}>
+        <div className={CLASS_NAME}>
```

Comment:
Thanks for fixing this, the way the string was interpolated before indeed did not make sense.

Reply:
You're welcome 😃